Libs/DICOM/Core/ctkDICOMDatabase.h

```diff
@@ -363,7 +363,7 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMDatabase : public QObject
   /// Insert lists of tags into the cache as a batch query operation
   Q_INVOKABLE bool cacheTags (const QStringList sopInstanceUIDs, const QStringList tags, const QStringList values);
   /// Remove all tags corresponding to a SOP instance UID
-  Q_INVOKABLE void removeCachedTags(const QString sopInstanceUID);
+  void removeCachedTags(const QString sopInstanceUID);
```

Comment:
This appears to be reverting recent commit https://github.com/commontk/CTK/commit/8f349212143bf23469a7e4c4f08617423a23538c. This diff here should be reverted to maintain that commit's changes.